### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by wussler**

### CODE REVIEW

File: draft-wussler-openpgp-pqc.md [hunk 1]
```diff
@@ -1581,6 +1583,32 @@ registry group at https://www.iana.org/assignments/pgp-parameters:
...
+- Added explicit key generation procedures.
+- Changed the key combination KMAC salt.
+- Mandated Parameter ID check in SPHINCS+ signature verification.
+- Fixed key share size for Kyber-768.
+- Added "Preliminaries" section.
+- Fixed IANA considerations.
+
+## draft-wussler-openpgp-pqc-02
+
+- Added the public key in the ECC key derivation function.
```
Comment (fluppe2) [inline draft-wussler-openpgp-pqc.md:1602]:
```suggestion
- Added the ephemeral and public key in the ECC key derivation function.
```

**Comment by fluppe2** [comment]
When did we allow KEMs for v4? Maybe this is also nice to see in the changelog.

**Comment by falko-strenzke** [comment]
> When did we allow KEMs for v4? Maybe this is also nice to see in the changelog.

That must have been in -02 since for that version it is listed that we allow PKESKv3 with ML-KEM composites. Allowing it for v4 keys was done at the same time.

**Comment by wussler** [comment]
Yes, I just listed v3 PKESK, but will add V4 keys

File: draft-wussler-openpgp-pqc.md [hunk 2]
```diff
@@ -1581,6 +1583,32 @@ registry group at https://www.iana.org/assignments/pgp-parameters:
...
+- Removed public key hash from key combiner.
+- Allowed v3 PKESKs with PQ algorithms, limiting them to AES symmetric ciphers
```
Comment (wussler) [inline draft-wussler-openpgp-pqc.md:1604]:
```suggestion
- Allowed v3 PKESKs and v4 keys with PQ algorithms, limiting them to AES
  symmetric ciphers
```

File: draft-wussler-openpgp-pqc.md [hunk 3]
```diff
@@ -1581,6 +1583,33 @@ registry group at https://www.iana.org/assignments/pgp-parameters:
...
+- Fixed key share size for Kyber-768.
+- Added "Preliminaries" section.
+- Fixed IANA considerations.
+
+## draft-wussler-openpgp-pqc-02
+
+- Added the ephemeral and public key in the ECC key derivation function.
+- Removed public key hash from key combiner.
+- Allowed v3 PKESKs and v4 keys with PQ algorithms, limiting them to AES
+  symmetric ciphers
```
Comment (wussler) [inline draft-wussler-openpgp-pqc.md:1605]:
```suggestion
  symmetric ciphers.
```

**Review by falko-strenzke — APPROVED** [review]

**Review by fluppe2 — APPROVED** [review]

**Pull request merged**